test/expression.txt

```diff
@@ -363,7 +363,7 @@ PromQL(
             MatchOp(NeqRegex),
             StringLiteral
           )
-        ),
```

Comment:
These changes are here just to align the expected and actual output as now it compares strings.